schema.graphql

```diff
@@ -56,15 +88,26 @@ enum EventName {
 }
 
 type EscrowEvent @entity {
+  "date payment set up"
   id: ID!
+  "network contract address"
   contractAddress: Bytes!
+  "is the reference data used to track the escrow event"
   reference: Bytes!
+  "smart contract that triggers when a payment is dispursed"
```

Comment:
Minor spelling error. dispersed should be disbursed. https://www.vocabulary.com/articles/chooseyourwords/disperse-disburse/#:~:text=disperse%2F%20disburse,want%20your%20money%20to%20disperse!